multimodal.py

```
@@ -0,0 +1,494 @@
+        torso_center = telemetry.get("torso_center")
+
+        if pose_detected and torso_center and face_center:
+            w = frame_dims[0]
+            center_dist = abs(torso_center[0] - (w // 2)) / (w * 0.5)
+            alignment_score = max(0.0, 1.0 - center_dist)
+            pose_score = 80.0 + (alignment_score * 12.0)
+            if hand_detected:
+                pose_score += 5.0
+        elif pose_detected:
+            pose_score = 82.0
+        elif face_detected:
```

Ruff (0.16.3)

[error] Undefined name `face_detected`

(F821)